app/Livewire/Server/Security/TerminalAccess.php

```
@@ -0,0 +1,85 @@
+
+            // Update the local property
+            $this->isTerminalEnabled = $this->server->settings->is_terminal_enabled;
+
+            $status = $this->isTerminalEnabled ? 'enabled' : 'disabled';
+            $this->dispatch('success', "Terminal access has been {$status}.");
+        } catch (\Throwable $e) {
+            return handleError($e, $this);
+        }
+    }
+
+    public function syncData(bool $toModel = false)
```

PHPMD (2.15.0)

The method syncData has a boolean flag argument $toModel, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)